lib/package.py

```diff
@@ -26,8 +26,6 @@
 
 CONF = config.get_config().CONF
 LOG = logging.getLogger(__name__)
-BUILD_DEPENDENCIES = "build_dependencies"
-DEPENDENCIES = "dependencies"
 
```

Comment:
Yes. I did this to be able to include dependencies in release notes, but that's not necessary.